operational-documentation/runbook.json

```diff
@@ -0,0 +1,31 @@
+{
+	"dataItemID": "origami-registry-ui",
+	"name": "Origami Registry UI",
+	"description": "Get information about Origami components, services, and repositories",
+	"serviceTier": "Bronze",
+	"lifecycleStage": "Requirements",
+	"primaryURL": "https://origami-registry-ui.ft.com",
+	"gitRepository": "https://github.com/Financial-Times/origami-registry-ui",
+	"hostPlatform": "Heroku",
+	"primaryContact": {
+		"contact": [{
+			"dataItemID": "origamisupport"
+		}]
+	},
+	"programme": {
+		"contact": [{
+			"dataItemID": "internalproducts"
```

Comment:
Are we under internal products?

Reply:
This is just what we have on all of our services at the mo. Adding to be consistent, but we should change all of them another time if we need to.